src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double side;
+
+    public Square(Color color, double side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double calculateArea() {
+        return side * side;
+    }
+
+    @Override
+    public void drawFigure() {
+        System.out.println("Figure: square, area: " + calculateArea() + " sq. units, side: "
+                + getSide() + " units, color: " + getColor());
```

Comment:
Magic numbers should be avoided. In this context, the string format for the figure's description could be extracted to a constant at the class level, which would make it easier to manage and understand.